src/GovUk.Education.ExploreEducationStatistics.Common/Model/TimeIdentifierCategory.cs

```diff
@@ -28,7 +28,7 @@ public enum TimeIdentifierCategory
         [EnumLabelValue("Week")]
         Week,
 
-        [EnumLabelValue("Other")]
-        CustomPeriod
+        [EnumLabelValue("Financial year part")]
+        FinancialYearPart
```

Comment:
I think there's a front end change needed in the form for creating a Release to use the label "Year" for the year field otherwise at the moment it will display a field label like "FinancialYearPart". It previously already had a case for "Other" which mapped to "Year" so by giving this category a label of "Other" it just worked before.